src/vivarium_gates_nutrition_optimization/constants/results.py

```diff
@@ -26,28 +26,20 @@
 
 THROWAWAY_COLUMNS = [f"{state}_event_count" for state in models.STATES]
 
-DEATH_COLUMN_TEMPLATE = "death_due_to_{CAUSE_OF_DEATH}_age_{AGE_GROUP}"
-YLLS_COLUMN_TEMPLATE = "ylls_due_to_{CAUSE_OF_DEATH}_age_{AGE_GROUP}"
-YLDS_COLUMN_TEMPLATE = "ylds_due_to_{CAUSE_OF_DISABILITY}_age_{AGE_GROUP}"
-PREGNANCY_STATE_PERSON_TIME_COLUMN_TEMPLATE = "pregnancy_{PREGNANCY_STATE}_person_time_age_{AGE_GROUP}_pregnancy_outcome_{PREGNANCY_OUTCOME}"
-PREGNANCY_TRANSITION_COUNT_COLUMN_TEMPLATE = "pregnancy_{PREGNANCY_TRANSITION}_event_count_age_{AGE_GROUP}_pregnancy_outcome_{PREGNANCY_OUTCOME}"
-
-MATERNAL_DISORDERS_PERSON_TIME_COLUMN_TEMPLATE = (
-    "maternal_disorders_{MATERNAL_DISORDERS_STATE}_person_time_age_{AGE_GROUP}_anemia_status_at_birth_{ANEMIA_STATUS_AT_BIRTH}"
-)
-MATERNAL_DISORDERS_TRANSITION_COUNT_COLUMN_TEMPLATE = (
-    "maternal_disorders_{MATERNAL_DISORDERS_TRANSITION}_event_count_age_{AGE_GROUP}_anemia_status_at_birth_{ANEMIA_STATUS_AT_BIRTH}"
-)
+DEATH_COLUMN_TEMPLATE = "MEASURE_death_due_to_{CAUSE_OF_DEATH}_AGE_GROUP_{AGE_GROUP}"
```

Comment:
Are these changes to the templates required by the new system? Or is this a side-request from RT or soemthing like that?

Reply:
Required - this is how the new system formats the output.

Reply:
Yes, that's the way the columns are structured in the `output.hdf` file